example/lib/named_routes.dart

```diff
@@ -160,11 +171,7 @@ class FamilyPage extends StatelessWidget {
             for (final p in family.people)
               ListTile(
                 title: Text(p.name),
-                onTap: () => context.go(context.namedLocation(
-                  'person',
-                  params: {'fid': family.id, 'pid': p.id},
-                  queryParams: {'qid': 'quid'},
```

Comment:
A bug @csells ? This is the type of thing that's missed when it's all strings, right?